### ISSUE TITLE
Proximity Based Blackout

### ISSUE THREAD

**Issue by alex-taffe** [Triggertrap/triggertrap-ios#24]
An option should be added to settings that allows for “proximity based blackout”. When enabled, when the phone is placed face down (triggers the proximity sensor), the screen should be turned off completely and all animations paused. This will significantly improve battery life. Once the phone is lifted, the screen will turn back on and animations will resume

**Comment by Ross-Gibson** [comment]
@alex-taffe I think we should have a discussion about this before implementing it. I'd like to try and avoid adding features that have edge cases, and the goal here is to reduce battery consumption - so we should think how best to address that instead.

**Comment by alex-taffe** [comment]
@Ross-Gibson I agree but when doing something like a full night time lapse, battery is of the essence. Once I fix #23 this may become a non issue, but I’ve had my phone die when using this app. If the display is off (phone obviously can’t fully sleep) that may help our case enough
